project_templates/stack_package/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change log
 
+## 2019-11-04
+
+Added a new `uses_dds` Cookiecutter variable.
+When enabled, this variable adds a block to the `tests/SConscript` file that adds the contents of the  `OSPL_URI`, `OPENSPLICE_LOC`, and `ADLINK_LICENSE` environment variables to the PATH of the SCons environment.
+These are necessary for T&S packages that work with DDS (the base technology used by SAL).
```

Comment:
These environment variables are unique to ADLink's OpenSplice DDS, so please consider "OpenSplice DDS" instead of "DDS" (here and in some of the comments below that I will call out).